### PR TITLE
Added R Data Visualizations lessons

### PR DESCRIPTION
**Pull request by Vidushi-Gupta**
Added the  following:
* Modified the main README.md of the 'data visualization' topic to include files from both R and Python.
* 'Visualizing Quantities' lesson in R on the same lines as the Python lesson. 

Committing this lesson for your review and suggestions.

As per the suggestions, I'll commit the other lesson soon too. 
Links to issue [#285 ](https://github.com/microsoft/Data-Science-For-Beginners/issues/285)

### CODE REVIEW

File: 3-Data-Visualization/R/09-visualization-quantities/README.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
+# Visualizing Quantities
+|![ Sketchnote by [(@sketchthedocs)](https://sketchthedocs.dev) ](https://github.com/microsoft/Data-Science-For-Beginners/blob/main/sketchnotes/09-Visualizing-Quantities.png)|
+|:---:|
+| Visualizing Quantities - _Sketchnote by [@nitya](https://twitter.com/nitya)_ |
+
+In this lesson you will explore how to use some of the many available R packages libraries to learn how to create interesting visualizations all around the concept of quantity. Using a cleaned dataset about the birds of Minnesota, you can learn many interesting facts about local wildlife. 
+## [Pre-lecture quiz]() TO BE LINKED
```
Comment (jlooper) [inline 3-Data-Visualization/R/09-visualization-quantities/README.md:7]: hi, you can link the standard quiz here

File: 3-Data-Visualization/R/09-visualization-quantities/README.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+  geom_bar(aes(y=MaxLength), stat="identity", position ="identity",  fill='blue') +
+  geom_bar(aes(y=MinLength), stat="identity", position="identity", fill='orange')+
+  coord_flip()
+```
+![super-imposed values](images/superimposed-values.png)
+
+## 🚀 Challenge
+
+This bird dataset offers a wealth of information about different types of birds within a particular ecosystem. Search around the internet and see if you can find other bird-oriented datasets. Practice building charts and graphs around these birds to discover facts you didn't realize.
+## [Post-lecture quiz]() To be linked
```
Comment (jlooper) [inline 3-Data-Visualization/R/09-visualization-quantities/README.md:209]: here too

File: 3-Data-Visualization/R/09-visualization-quantities/README.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+
+## Review & Self Study
+
+This first lesson has given you some information about how to use `ggplot2`to visualize quantities. Do some research around other ways to work with datasets for visualization. Research and lookout for datasets that you could visualize using other packages like [Lattice](https://stat.ethz.ch/R-manual/R-devel/library/lattice/html/Lattice.html) and [Plotly](https://github.com/plotly/plotly.R#readme)
+
+## Assignment
```
Comment (jlooper) [inline 3-Data-Visualization/R/09-visualization-quantities/README.md:215]: you can link an R assignment here

**Review by jlooper — CHANGES_REQUESTED** [review]
hi, this is looking nice! I don't know what the covid dataset is about, so that can be removed. I recommend keeping this in draft until all the parts are added (assignments, quiz links, and all text). But very nice!

File: 3-Data-Visualization/R/09-visualization-quantities/README.md (hunk 1 above)
Comment (Vidushi-Gupta) [reply 3-Data-Visualization/R/09-visualization-quantities/README.md:7]: Just wanted to confirm if the standard quiz = the quiz on the Python Readme for the same topic?

**Comment by Vidushi-Gupta** [comment]
My bad, I forgot to remove the COVID dataset. Yes, I'll link in the quizzes and yes, we can leave this as a draft. I'll push the next lesson very soon. Thank you for your review!

**Comment by Vidushi-Gupta** [comment]
Hey, @jlooper. The lessons on the first two topics are ready. Please let me know your suggestions and views.
On merging of this PR, @jasleen101010 (also assigned on the same issue) would be committing the lessons for the other three topics. Thanks!

File: 3-Data-Visualization/R/09-visualization-quantities/README.md (hunk 1 above)
Comment (jlooper) [reply 3-Data-Visualization/R/09-visualization-quantities/README.md]: hi, @Vidushi-Gupta sorry I didn't see this. Yes, I think we have to keep this the same quiz

**Comment by jlooper** [comment]
awesome, @Vidushi-Gupta - thank you for your hard work! @jasleen101010 is there an estimated date on the other topics? thanks again!

**Comment by jasleen101010** [comment]
> awesome, @Vidushi-Gupta - thank you for your hard work! @jasleen101010 is there an estimated date on the other topics? thanks again!

Hi @jlooper! I am hoping to make a PR with my topics by the 28th. Hope that's okay  :)

**Comment by jasleen101010** [comment]
> Hey, @jlooper. The lessons on the first two topics are ready. Please let me know your suggestions and views.
> On merging of this PR, @jasleen101010 (also assigned on the same issue) would be committing the lessons for the other three topics. Thanks!

Hey, @jlooper ! Just dropping in a reminder to merge @Vidushi-Gupta 's PR. Since, she has done the first 3 lessons and created the file structure for the R lessons, I will be raising my PR only after hers is merged. Thanks in advance!

**Review by jlooper — APPROVED** [review]
awesome! thank you!

**Comment by jlooper** [comment]
thank you ladies!!

**Comment by jasleen101010** [comment]
Hi @jlooper ! I seem to have hit a roadblock with one of the graphs in lesson 2. I have asked Jasmine for her help. I will be raising a PR after her recommendations. Just keeping you in the loop, thanks :)

**Pull request merged**